components/LandingPage.js

```diff
@@ -1,22 +1,26 @@
 import React from "react";
-import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+import { View, Text, StyleSheet, TouchableOpacity, ImageBackground } from "react-native";
 import { Header } from 'react-native-elements';
 import normalize from 'react-native-normalize';
 
 const LandingPage = ({ navigation }) => {
   return (
     <View style={styles.container}>
       <Header
-        containerStyle={{ backgroundColor: 'black', justifyContent: 'space-around', paddingBottom: 10}}
+        containerStyle={{ backgroundColor: '#483D8B', justifyContent: 'space-around', paddingBottom: 10}}
         centerComponent={{ text: 'Never Alone', style: { color: '#fff', fontSize: normalize(30), fontWeight: "bold" } }}
       />
       <View style={styles.box}>
         <Text style={styles.text}>Choose Your Role:</Text>
         <TouchableOpacity style={styles.careTakerButton} onPress={() => navigation.navigate('Caretaker')} >
-          <Text style={styles.buttonText}>Caretaker</Text>
+          <ImageBackground source={require('../images/caretaker.jpg')} style={styles.backgroundImage} >
+            <Text style={styles.buttonText}>Caretaker</Text>
+          < /ImageBackground>
         </TouchableOpacity>
```

Comment:
I think this would be a cool refactor into it's own component. Especially if we wanted to have this style of button in a larger app where it would be used in a lot more places.